carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -39,11 +42,13 @@ namespace carma_wm_ctrl
 using std::placeholders::_1;
 
 
-WMBroadcaster::WMBroadcaster(const PublishMapCallback& map_pub, const PublishMapUpdateCallback& map_update_pub, std::unique_ptr<TimerFactory> timer_factory)
-  : map_pub_(map_pub), map_update_pub_(map_update_pub), scheduler_(std::move(timer_factory))
+WMBroadcaster::WMBroadcaster(const PublishMapCallback& map_pub, const PublishMapUpdateCallback& map_update_pub, const RouteMsgCallback& route_callmsg_pub,
+std::unique_ptr<TimerFactory> timer_factory)
+  : map_pub_(map_pub), map_update_pub_(map_update_pub), route_callmsg_pub_(route_callmsg_pub), scheduler_(std::move(timer_factory))
 {
   scheduler_.onGeofenceActive(std::bind(&WMBroadcaster::addGeofence, this, _1));
   scheduler_.onGeofenceInactive(std::bind(&WMBroadcaster::removeGeofence, this, _1));
+  std::bind(&WMBroadcaster::routeCallbackMessage, this, _1);
```

Comment:
This line is doing nothing because the bound object is not stored and can be deleted.